### PR TITLE
[BUG] make macro more resilient to missing data

### PR DESCRIPTION
**Pull request by emekdahl**
[Slack Thread](https://vouchinc.slack.com/archives/C015VL132UC/p1670334619593089)

[JIRA Ticket](https://vouchinc.slack.com/archives/C015VL132UC/p1670334619593089)

### CODE REVIEW

File: macros/object_pivot.sql [hunk 1]
```diff
@@ -12,7 +12,11 @@
```
Comment (emekdahl) [inline macros/object_pivot.sql:12]: this is the basic idea but I need to touch base with @jennaallen on how to test locally

File: macros/object_pivot.sql [hunk 2]
```diff
@@ -12,7 +12,11 @@
 
     select
     {%- for ic in include_columns %}
-        {{ ic }},
+        {%- if len(keys) > 0 or (keys == [] and not loop.last) -%}
+            {{ ic }},
```
Comment (avyfain) [inline macros/object_pivot.sql]: Should we make this always be `{{ ic }}` and only add the comma on the opposite of this condition so we don't need if/else and just if? 

Even better, could we do something like a `',\n'.join(blah)` in python? I don't know enough jinja to be honest.

Comment (emekdahl) [reply macros/object_pivot.sql]: we're keeping track of two different lists here, which is why i made it longer in the hope of it being more readable.

File: macros/object_pivot.sql (hunk 1 above)
Comment (emekdahl) [reply macros/object_pivot.sql:12]: I figured out how to test. This now generates the code: 

```-- depends_on: emilyekdahl.stg_policy_events__location_specific_rater_inputs_flattened
select policy_id, event_id,location_id
    from emilyekdahl.stg_policy_events__location_specific_rater_inputs_flattened
```

Comment (emekdahl) [reply macros/object_pivot.sql:12]: in vouch-dwh packages.yml, the package needs to be updated to: 
```
  - git: "https://github.com/svinstech/dbt-variant-utils.git"
    revision: "99674b408403d41b0db8098e4de6c9e0582d0c08"
```
and then run the command `dbt deps && dbt run --select +stg_policy_events__location_specific_rater_inputs` to test that the macro is working

Comment (nzewail) [reply macros/object_pivot.sql:12]: did you run that test?

**Review by nzewail — APPROVED** [review]
LGTM!

File: macros/object_pivot.sql (hunk 1 above)
Comment (emekdahl) [reply macros/object_pivot.sql:12]: yes i did, why did it not work for you?

**Pull request merged**